### ISSUE TITLE
ktlink remove assignments when assignment in new line

### ISSUE THREAD

**Issue by levirs565** [pinterest/ktlint#1066]
I have this code:
```kotlin
    override fun getLaunches(
        scope: CoroutineScope,
        filterByName: String,
        filterByState: LaunchEntity.State?,
        sortBy: LaunchSortBy
    ): Flow<ResultState<PagingData<LaunchEntity>>>
        = mRepository.getLaunches(scope, filterByName, filterByState, sortBy)


    override fun getFavoriteLaunches(scope: CoroutineScope): Flow<PagingData<LaunchEntity>>
        = mRepository.getFavoriteLaunches(scope)

    override fun getLaunchById(id: String): Flow<LaunchEntity>
        = mRepository.getLaunchById(id)

```
When i format using ktlint assignment operator is removed.

## Expected Behavior
Assignment operator not removed.

## Observed Behavior
Assignment operator is removed.

## Steps to Reproduce
1. Save code to `test.kt`:
    ```kotlin
    fun getName()
        = "Test"

    fun main() {
        println(getName())
    }
    ```
2. Run ktlint with this command:
    ```
    ktlint -F test.kt
    ```
3. Now code changed like this:
    ```kotlin
    fun getName()
    "Test"

    fun main() {
        println(getName())
    }
    ````
## Your Environment
* Version of ktlint used: 0.40.0
* Commandline using Git For Windows
* Operating System and version: Windows 10 1909
* OpenJDK Runtime Environment (build 1.8.0_275-b01)
    Eclipse OpenJ9 VM (build openj9-0.23.0, JRE 1.8.0 Windows 10 amd64-64-Bit Compressed References 20201112_897 (JIT enabled, AOT enabled)
    OpenJ9   - 0394ef754
    OMR      - 582366ae5
    JCL      - b52d2ff7ee based on jdk8u275-b01)

**Comment by t-kameyama** [comment]
I think this issue has been resolved by #1041.

**Comment by Tapchicoma** [comment]
Yes, it is resolved by linked PR :+1:
